src/components/Button/Button.jsx

```diff
@@ -114,7 +114,7 @@ const Button = createClass({
 
 		return (
 			<button
-				{...passThroughs}
+				{...omitProps(passThroughs, Button, ['callbackId'])}
```

Comment:
Yeah agreed. I'm not keen on the pattern either. I recently had a little discussion on twitter about this. One solution was to use a memoized version of `partial`. I'm not sure I completely understand the implications of it, but it's food for thought.

https://twitter.com/davidlymanning/status/715993185403740160